src/modules/tasks.nix

```diff
@@ -123,7 +123,7 @@ in
       description = "The generated tasks.json file.";
     };
     task.package = lib.mkOption {
-      type = config.lib.types.output;
+      type = types.package;
```

Comment:
@domenkozar is this a typo? this gets picked up by `devenv build`, which is kinda weird for end-users to see.
I guess a related question is whether we should be recursively looking for outputs in the first place?

Reply:
I was thinking at the time, that `devenv build` should build all packages used inside the environment. Maybe that's where the line should be drawn, most users would expect it to build only their own.